packages/@aws-cdk/aws-s3/lib/bucket.ts

```diff
@@ -1310,6 +1318,8 @@ export class Bucket extends BucketBase {
   private readonly metrics: BucketMetrics[] = [];
   private readonly cors: CorsRule[] = [];
   private readonly inventories: Inventory[] = [];
+  private readonly enforceSSL?: boolean;
+  private readonly blockPublicAccess: BlockPublicAccess | undefined;
```

Comment:
Why do you need these? Seems like you can just do:

```ts
if (props.enforceSSL) {
  this.enforceSSLStatement();
}
```

Reply:
Ah yeah thats a good catch. I will go ahead and make that change to both of those. Thanks!